config/locales/en.yml

```diff
@@ -168,7 +168,6 @@ en:
       previous_strikes_description_html:
         one: This account has <strong>one</strong> strike.
         other: This account has <strong>%{count}</strong> strikes.
-        zero: This account is <strong>in good standing</strong>.
```

Comment:
This wouldn't show up anyway as `previous_strikes_description_html` is only ever used when there are warnings to display, so `count` would never be 0.